packages/components/src/components/input/input.tsx

```diff
@@ -28,11 +28,13 @@ type TInputProps = {
     name?: string;
     onBlur?: (e: React.FocusEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
     onChange?: (e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+    onKeyUp?: (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+    onKeyDown?: (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
     onFocus?: (e: React.FocusEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
     onPaste?: (e: React.ClipboardEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
     placeholder?: string;
     required?: boolean;
-    trailing_icon?: React.ReactElement;
+    trailing_icon?: React.ReactElement | null;
```

Comment:
@rostislav-deriv It was already optional, Any specific reason for adding `null` too? 🤔

Reply:
This is because of this https://github.com/binary-com/deriv-app/pull/7363/files/0df8ae59b0edea0016b5cea7f10944e90f8de93f#diff-f9654ab9677879818ad0d46350d6373f45633d516aa831d8e74ecca4b9222cc6R62-R70

The on-false operand in the ternary operator expression could be `undefined` but `null` is more suitable (source: George). It really seems more logical in a situation where we need rendering nothing to return the code equivalent of nothing (`null`), rather than code equivalent of "I dunno" (`undefined`).

Also, the `trailing_icon` prop could be alternatively defined as `ReactNode`, but I thought it's better to allow one more possible option (`null`), rather than lots of them (`number`, `string`, virtually anything renderable, as `ReactNode` is exactly for that).